presentation/src/main/java/com/yapp/bol/presentation/view/match/game_select/GameSelectFragment.kt

```diff
@@ -0,0 +1,54 @@
+package com.yapp.bol.presentation.view.match.game_select
+
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.navigation.fragment.findNavController
+import com.yapp.bol.presentation.R
+import com.yapp.bol.presentation.databinding.FragmentGameSeleteBinding
+import com.yapp.bol.presentation.model.GameItem
+import dagger.hilt.android.AndroidEntryPoint
+
+@AndroidEntryPoint
+class GameSelectFragment : Fragment() {
+
+    private var _binding: FragmentGameSeleteBinding? = null
+    private val binding get() = checkNotNull(_binding)
+
+    private val recyclerAdapter = GameSelectAdapter { gameName ->
+        val bundle = Bundle().apply {
+            putString(GAME_NAME, gameName)
+        }
+        findNavController().navigate(
+            R.id.action_gameSelectFragment_to_memberSelectFragment, bundle
+        )
+    }
+
+    override fun onCreateView(
+        inflater: LayoutInflater, container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        _binding = FragmentGameSeleteBinding.inflate(inflater, container, false)
+        return _binding?.root
+    }
```

Comment:
p5 : 
```suggestion
   ): View = FragmentGameSeleteBinding.inflate(inflater, container, false).apply {
        _binding = this
    }.root
```
개인적으로 요렇게 하는것을 선호합니다

Reply:
오 괜찮은 방법이네요 !  근데 개인적으로는 가독성이 조금 떨어지는 느낌...